build_debian.sh

```diff
@@ -307,6 +306,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     tcptraceroute           \
     mtr-tiny                \
     locales                 \
+    netbase                 \
+    libopts25               \
```

Comment:
i do not think these changes are needed since when we install ntp, we will install missing dependencies.

Reply:
Let me test it out.

Reply:
removed..

Reply:
brought back this change to avoid the build failure.